blockmanager.go

```diff
@@ -902,6 +939,13 @@ func (b *blockManager) haveInventory(invVect *wire.InvVect) (bool, error) {
 // handleInvMsg handles inv messages from all peers.
 // We examine the inventory advertised by the remote peer and act accordingly.
 func (b *blockManager) handleInvMsg(imsg *invMsg) {
+	peer, exists := b.peers[imsg.peerID]
+	if !exists {
+		bmgrLog.Errorf("Received inv message from unknown peer %d (%d)",
```

Comment:
Oops, this is leftover from when I was debugging the unknown peer warnings. Will remove.